client/src/clientConfig.ts

```diff
@@ -32,31 +31,25 @@ class ConflictLogger implements ConflictListener {
 
 const link = split(
   ({ query }) => {
-    // @ts-ignore
-    const { kind, operation} = getMainDefinition(query);
+    const { kind, operation} : any = getMainDefinition(query);
     return kind === 'OperationDefinition' && operation === 'subscription';
   },
   wsLink,
   httpLink,
 );
 
-export const clientConfig = {
-  link,
-  cache: new InMemoryCache(),
-  resolvers: {
+const cache =  new InMemoryCache({
+  cacheRedirects: {
     Query: {
-      // @ts-ignore
-      getTask: async (_, variables, { cache, getCacheKey }) => {
-        const id = getCacheKey({ __typename: 'Task', id: variables.id });
-        const data = cache.readFragment({ id, fragment: FRAGMENT });
-        console.log('here');
-        return {
-          ...data,
-          __typename: 'Task'
-        };
-      }
+      getTask: (_, args, { getCacheKey }) =>
```

Comment:
This looks like something we can also generate based on model types

Reply:
Agreed!